package.xml

```diff
@@ -38,12 +38,14 @@
 
   <build_depend>git</build_depend>
   <build_depend>google-mock</build_depend>
-  <build_depend>python-sphinx</build_depend>
+  <build_depend>gtest</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 2">python-sphinx</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 3">python3-sphinx</build_depend>
```

Comment:
If this goes to master, does it work with ROS 1?

Reply:
> If this goes to master, does it work with ROS 1?

Yeah, it should.  Both Melodic and Noetic support package format 3, and thus support `build_depend` with a condition.  At least in my testing on Melodic, I can both build the package and install the (Python 2) dependencies via `rosdep`.

Reply:
cool, I just haven't seen this condition thing yet.